helper-cli/src/main/kotlin/commands/scanstorage/DeleteCommand.kt

```diff
@@ -113,6 +113,10 @@ internal class DeleteCommand : CliktCommand(
             }
 
             println("Would delete ${results.size} scan result(s).")
+
+            if (log.delegate.isDebugEnabled) {
+                results.map { it[ScanResults.identifier] }.forEach(log::debug)
+            }
```

Comment:
This would not work anymore with the `count()` from my previous comment. Instead you could use this query to not download the scan results but only the identifiers: `ScanResults.slice(ScanResults.identifier).select { condition }`